service/kernel/error_handling.cpp

```diff
@@ -950,6 +950,13 @@ void ErrorMessageCollection::parseResourceFile()
 
     add(NoErrorMessageFound, "NoErrorMessageFound");
     add(ErrorMethodNotImplemented, "Method is not implemented");
+
+    add(ErrorIncorrectOffset, "Incorrect offset");
+    add(ErrorAdagradIncorrectMaxNumberOfIterations, "Incorrect maximum number of iterations value in adagrad solver parameter");
+    add(ErrorLbfgsIncorrectMaxNumberOfIterations, "Incorrect maximum number of iterations value in lbfgs parameter");
+    add(ErrorSgdIncorrectMaxNumberOfIterations, "Incorrect maximum number of iterations value in sgd parameter");
```

Comment:
is it agreed name for specific algorithm? why don`t we use the same error for all of them?

Reply:
These name are approved by Averbukh M.